homeassistant/components/homewizard_energy/sensor.py

```diff
@@ -0,0 +1,228 @@
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=DEVICE_CLASS_SIGNAL_STRENGTH,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_ENERGY_IMPORT_T1_KWH,
+        name="Total Power Import T1",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_ENERGY_IMPORT_T2_KWH,
+        name="Total Power Import T2",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_ENERGY_EXPORT_T1_KWH,
+        name="Total Power Export T1",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_ENERGY_EXPORT_T2_KWH,
+        name="Total Power Export T2",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_W,
+        name="Active Power",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_L1_W,
+        name="Active Power L1",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_L2_W,
+        name="Active Power L2",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_L3_W,
+        name="Active Power L3",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_GAS_M3,
+        name="Total Gas",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        device_class=DEVICE_CLASS_GAS,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_GAS_TIMESTAMP,
+        name="Gas Timestamp",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+    ),
+)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Config entry example."""
+    energy_api = hass.data[DOMAIN][entry.data["unique_id"]][CONF_API]
+    coordinator = hass.data[DOMAIN][entry.data["unique_id"]][COORDINATOR]
+
+    # Fetch initial data so we have data when entities subscribe
+    await coordinator.async_refresh()
```

Comment:
this should already have happened at
https://github.com/home-assistant/core/blob/a49d707e3042e3862025bc5fdea7831e7fa0c676/homeassistant/components/homewizard_energy/__init__.py#L72

Reply:
Thanks, fixed in b945a09e690a5c995742befbb4806d31da51a6ca